nixos/modules/services/editors/infinoted.nix

```diff
@@ -124,33 +125,34 @@ in {
 
         wantedBy = [ "multi-user.target" ];
         after = [ "network.target" ];
-        path = [ cfg.package ];
 
         serviceConfig = {
           Type = "simple";
           Restart = "always";
           ExecStart = "${cfg.package}/bin/infinoted-0.6 --config-file=/var/lib/infinoted/infinoted.conf";
+          User = cfg.user;
+          Group = cfg.group;
+          ExecStartPre = "+" + pkgs.writeScript "infinoted-setup" ''
```

Comment:
In what version of systemd is that? It's not mentioned in the documentation, and here that syntax just causes systemd to fail when trying to load the unit.